server/helpers/sendEmail.js

```diff
@@ -0,0 +1,25 @@
+import sgMail from '@sendgrid/mail';
+
+sgMail.setApiKey(process.env.SENDGRID_API_KEY);
+
+/**
+ * This function sends a mail to user
+ * @param {object} user object containing user email and uuid
+ * @param {object} message object containing message subject and content body
+ * @returns {object} error object, null if successful
+ */
+const sendEmail = (user, message) => {
+  if (!user.email) {
```

Comment:
Shouldn't you also check for the message body and also the subject? That way we are sure we are not sending email without a subject.

Reply:
Yes, this is true, I'll do that